security/network_bound_disk_encryption/nbde-about-disk-encryption-technology.adoc

```diff
@@ -0,0 +1,30 @@
+// CNF-2127 assembly
+[id="nbde-about-disk-encryption-technology"]
+= About disk encryption technology
+include::modules/common-attributes.adoc[]
+:context: nbde-implementation
```

Comment:
The [assembly metadata](https://github.com/openshift/openshift-docs/blob/main/contributing_to_docs/doc_guidelines.adoc#assembly-file-metadata) section of the docs guidelines say that the context need to be unique for each assembly.  "about-nbde" is prolly OK.

I don't plan to point that out for the other assemblies.